invenio_vocabularies/datastreams/readers.py

```diff
@@ -54,3 +55,29 @@ def read(self):
                 if member.isfile() and match:
                     content = archive.extractfile(member).read()
                     yield content
+
+
+class SimpleHTTPReader(BaseReader):
```

Comment:
IMO this is now more flexible, allows for:
- different content-types
- different endpoints, as long as the url (origin) is a string to be formatted. I did it this way because concatenating might not always work (I bet some endpoints might not have the id at the end...)
- `id` and `ids` was not to force an array, I think it is a "prettier" API this way, accepting one or the other (favouring `ids`).